index.html

```diff
@@ -246,7 +246,7 @@ <h1>Let's chat</h1>
                     <input type="email" id="email" name="email" class="email1" placeholder="Email" required>
                     <textarea name="message" id="message" class="text" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
                     <span class="error" aria-live="polite"></span>
-                    <button type="submit" class="btn" onclick="">Submit</button>
+                    <button type="submit" class="btn" onclick="content();">Submit</button>
                     
                 </form>
             </div>    
```

Comment:
- [ ] Good job 👍🏽 implementing the localstorage feature. But I noticed that your input data are saved when you submit the form. They should be submitted after filling out the form (before submission). One way to achieve this is by using the [blur event](https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event). A usage example in [W3C](https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_onblur).

[OPTIONAL] Kindly make your name input required. This should have been done in a previous activity

Reply:
@fobadara Well I did not give the input required trait as I have different name fields for the mobile and desktop version. If I make them required, the form would not submit for both of the versions. Kindly share with me any specific solution that comes to your mind.

Reply:
@fobadara Nonetheless, the name tags in both the versions is required just like the remaining elements of the form.